packer.gradle

```diff
@@ -0,0 +1,61 @@
+import static org.opendatakit.aggregate.gradle.Util.setPropertiesValue
+
+task packerClean() {
+  delete 'build/packer'
+}
+
+task packerDownloadPacker() {
+  doLast {
+    if (!file('packer/packer').exists()) {
+      println("Downloading Packer binary")
+      download {
+        src packerZip
+        dest 'packer/packer.zip'
+      }
+      "unzip packer/packer.zip -d packer".execute()
+      "rm packer/packer.zip".execute()
+    } else
+      println("Packer 1.1.3 linux x64 binary is already present")
+  }
+}
+
+
+task packerPrepareProject(dependsOn: [packerClean, packerDownloadPacker], type: Copy) {
+  into "${buildDir}/packer"
+  from fileTree(dir: 'packer')
+
+  into('ansible/roles/aggregate/files/ROOT') {
+    from zipTree("${buildDir}/libs/${archivesBaseName}-${version}.war")
+  }
+
+  doLast {
```

Comment:
I like `aggregate` as username password and schema because `odk` is a bit vague.